{"uid":"4b4757e66a1912dae1a509f688f20b0f","children":[{"name":"Test defects","children":[{"name":"Unable to provision, see the following errors:\n\n1) Error in custom provider, org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Response code 500. Message: An unknown server-side error occurred while processing the command. Original error: Appium Settings app is not running after 30000ms \nHost info: host: 'Mac-1713820000169.local', ip: '10.213.1.93'\nBuild info: version: '4.19.1', revision: 'abe0ee07dc'\nSystem info: os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '12.7.4', java.version: '11.0.22'\nDriver info: io.appium.java_client.android.AndroidDriver\nCommand: [null, newSession {desiredCapabilities=Capabilities {adbExecTimeout: 60000, androidInstallTimeout: 120000, app: /Users/runner/work/aquality..., appActivity: io.cloudgrey.the_app.MainAc..., appPackage: io.cloudgrey.the_app, automationName: UiAutomator2, deviceName: Android Emulator, platformName: ANDROID, uiautomator2ServerInstallTimeout: 120000}, capabilities=[{appium:uiautomator2ServerInstallTimeout=120000, appium:app=/Users/runner/work/aquality-appium-mobile-java-template/aquality-appium-mobile-java-template/aquality-appium-mobile-template/src/main/resources/applications/TheApp-v1.10.0.apk, appium:androidInstallTimeout=120000, appium:appActivity=io.cloudgrey.the_app.MainActivity, appium:deviceName=Android Emulator, appium:automationName=UiAutomator2, appium:adbExecTimeout=60000, appium:appPackage=io.cloudgrey.the_app, platformName=ANDROID}]}]\nCapabilities {adbExecTimeout: 60000, androidInstallTimeout: 120000, app: /Users/runner/work/aquality..., appActivity: io.cloudgrey.the_app.MainAc..., appPackage: io.cloudgrey.the_app, automationName: UiAutomator2, deviceName: Android Emulator, platformName: ANDROID, uiautomator2ServerInstallTimeout: 120000}\n  at aquality.selenium.core.applications.AqualityModule.configure(AqualityModule.java:41)\n  while locating aquality.selenium.core.applications.IApplication\n\n1 error","children":[{"name":"I try to login with invalid credentials","uid":"2315fec539627800","parentUid":"949d1f107ebb264e1b6555a6eb2746df","status":"broken","time":{"start":1713820622067,"stop":1713821107365,"duration":485298},"flaky":false,"newFailed":false,"newPassed":false,"newBroken":false,"retriesCount":0,"retriesStatusChange":false,"parameters":["ANDROID"],"tags":["demo"]}],"uid":"949d1f107ebb264e1b6555a6eb2746df"}],"uid":"bdbf199525818fae7a8651db9eafe741"}],"name":"categories"}